language/it/movemessage.php

```diff
@@ -17,5 +17,6 @@
 }
 
 $lang = array_merge($lang, array(
-	'MOVED_MESSAGE'				=> 'Spostato dal forum <strong>%1$s</strong> al forum <strong>%2$s</strong> da <strong>%4$s</strong> il %3$s',
+<<<<<<< HEAD
```

Comment:
That is not supposed to be here! Please check your code and remove those leftovers.